pandas/tests/reductions/test_reductions.py

```diff
@@ -1060,6 +1060,12 @@ def test_min_max_skipna(self, skipna):
             assert np.isnan(_min)
             assert np.isnan(_max)
 
+    def test_min_max_empty(self):
+        cat = Series(Categorical([], categories=list("ABC"), ordered=True))
+
+        assert isna(cat.min())
+        assert isna(cat.max())
```

Comment:
I think this should be ``assert cat.max() is np.nan`` instead. E.g. ``pd.isna(None)`` returns True, but should fail in this test.

Reply:
`np.nan` is usually, though not always a singleton. So `isinstance(result, float) and `np.isnan(result)` should do it.

Reply:
Ok, makes sense.